monitoring/mongodb/dashboard.py

```diff
@@ -1136,9 +1528,10 @@ def mongodb_state_timeline(title, expr, description="", mappings=None, **kwargs)
             [
                 Template(
                     dataSource=DATASOURCE,
-                    name="job",
-                    label="Job",
-                    query='label_values(mongodb_up{namespace="zenko"},job)',
+                    name="mongodb_instance_type",
+                    label="MongoDB instance type",
+                    query='label_values(mongodb_up{namespace="${namespace}"}, job)',
```

Comment:
Should still rely on the `job` template parameter provided by ZKOP, to ensure we return only the expected instance of mongo:

```suggestion
                    query='label_values(mongodb_up{namespace="${namespace}", job=~"^${job}"}, job)',
```